gnovm/pkg/gnomod/pkg.go

```diff
@@ -0,0 +1,156 @@
+package gnomod
+
+import (
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+)
+
+type pkg struct {
```

Comment:
I'm personally very much exposing unexported types through exported methods. I would say export this type; the package user cannot modify its fields anyway as they are unexported.

Reply:
💯 

Will export.

Reply:
Resolved by: ee268b849611662e7dc371cdd9d367f1d8ec1501